docs/apps/geoconda.md

```diff
@@ -51,22 +51,23 @@ Additionally geoconda includes:
     list.
     
 !!! note
-    If you want to use Spyder, ncview or other tools with graphical user interfaces, you should connect to Puhti using NoMachine and [start an interactive session](../computing/running/interactive-usage.md) for best performance. You can also [edit and run Python code remotely](../support/tutorials/remote-dev.md) with some code editors like Visual Studio Code
+    If you want to use Spyder, ncview or other tools with graphical user interfaces,connect to Puhti web interface and start a [remote desktop](../computing/webinterface/desktop.md) or [Visual Studio Code](../computing/webinterface/vscode.md). You can also [edit and run Python code remotely](../support/tutorials/remote-dev.md) with some code editors like Visual Studio Code
 
 Python has multiple packages for parallel computing, for example
 **multiprocessing**, **joblib** and **dask**. In our [Puhti Python examples](https://github.com/csc-training/geocomputing/tree/master/python/puhti) there are examples how to utilize these different parallelisation libraries.
 
-(If you think that some important GIS package for Python is missing from
-here, you can ask for installation from servicedesk@csc.fi.)
+(If you think that some important GIS package for Python is missing from here, you can ask for installation from servicedesk@csc.fi.)
 
 
 ## Available
 
 The `geoconda` module is available in Puhti:
 
-* 3.8.8 (Singularity installation with wrappers, so should be hidden from users in normal use-cases)
-* 3.8 
-* 3.7 
+* 3.8.8 (Singularity installation with wrappers, the usage of singularity container should not be visible in normal use-cases.)
+* 3.8-deprecated 
+* 3.7-deprecated 
+
+3.8 and 3.7 are depricated, because these are normal conda installations, which causes heavy load to Puhti disk system.
```

Comment:
```suggestion
3.8 and 3.7 are deprecated, because these are normal conda installations, which causes heavy load to Puhti disk system.
```

Reply:
I wonder if "because these are normal conda installations", confuses the user more than it resolves?